### ISSUE TITLE
feat: add celenium post a blob tutorial

### ISSUE THREAD

**Issue by jcstein** [celestiaorg/docs#1548]
a quick tutorial on posting a blob with celenium

**Comment by jcstein** [comment]
also, IMHO should be done with Arabica or Mocha for accessibility
